database-plugins/src/main/java/co/cask/hydrator/plugin/db/batch/sink/DBSink.java

```diff
@@ -51,10 +51,7 @@
 import java.sql.ResultSet;
 import java.sql.ResultSetMetaData;
 import java.sql.Statement;
-import java.util.ArrayList;
```

Comment:
Does it still compile?? I don't see any `import java.util.Map` which is used in line 148.